src/overlays/actors/ovl_En_Skb/z_en_skb.c

```diff
@@ -245,25 +246,25 @@ void func_80AFCFF0(EnSkb* this, PlayState* play) {
     }
 }
 
-void func_80AFD0A4(EnSkb* this) {
+void EnSkb_SetupWalkForward(EnSkb* this) {
     Animation_Change(&this->skelAnime, &gStalchildWalkingAnim, 0.96000004f, 0.0f,
                      Animation_GetLastFrame(&gStalchildWalkingAnim), ANIMMODE_LOOP, -4.0f);
-    this->unk_280 = 4;
-    this->unk_288 = 0;
+    this->actionState = ACTION_STATE_WALKING;
+    this->headlessWalkDirectionOffset = 0;
     this->actor.speedXZ = this->actor.scale.y * 160.0f;
-    EnSkb_SetupAction(this, EnSkb_Advance);
+    EnSkb_SetupAction(this, EnSkb_WalkForward);
 }
 
-void EnSkb_Advance(EnSkb* this, PlayState* play) {
+void EnSkb_WalkForward(EnSkb* this, PlayState* play) {
     s32 thisKeyFrame;
     s32 prevKeyFrame;
     f32 playSpeed;
     Player* player = GET_PLAYER(play);
 
-    if ((this->unk_283 != 0) && ((play->gameplayFrames & 0xF) == 0)) {
-        this->unk_288 = Rand_CenteredFloat(50000.0f);
+    if ((this->bodybreakState != 0) && ((play->gameplayFrames & 0xF) == 0)) {
+        this->headlessWalkDirectionOffset = Rand_CenteredFloat(50000.0f);
     }
-    Math_SmoothStepToS(&this->actor.shape.rot.y, (this->actor.yawTowardsPlayer + this->unk_288), 1, 0x2EE, 0);
+    Math_SmoothStepToS(&this->actor.shape.rot.y, (this->actor.yawTowardsPlayer + this->headlessWalkDirectionOffset), 1, 0x2EE, 0);
```

Comment:
this file needs formatting

We use `clang-format` (preferably `clang-format-11`) and `clang-tidy` for automatic code formatting
The formatter can be run on all .c files with `./format.sh` (script at the repo's root)
or on a single file `./format.sh src/overlays/actors/ovl_En_Skb/z_en_skb.c`